bin/MadGraph5_aMCatNLO/cards/production/13TeV/WPrime_tb/WPrime_tb_run_card.dat

```diff
@@ -0,0 +1,270 @@
+ 0   = ptl3min ! minimum pt for the third lepton in pt
+ 0   = ptl4min ! minimum pt for the fourth lepton in pt
+ -1  = ptl1max ! maximum pt for the leading lepton in pt
+ -1  = ptl2max ! maximum pt for the second lepton in pt
+ -1  = ptl3max ! maximum pt for the third lepton in pt
+ -1  = ptl4max ! maximum pt for the fourth lepton in pt
+#*********************************************************************
+# Control the Ht(k)=Sum of k leading jets                            *
+#*********************************************************************
+ 0   = htjmin ! minimum jet HT=Sum(jet pt)
+ -1  = htjmax ! maximum jet HT=Sum(jet pt)
+ 0   = ihtmin  !inclusive Ht for all partons (including b)
+ -1  = ihtmax  !inclusive Ht for all partons (including b)
+ 0   = ht2min ! minimum Ht for the two leading jets
+ 0   = ht3min ! minimum Ht for the three leading jets
+ 0   = ht4min ! minimum Ht for the four leading jets
+ -1  = ht2max ! maximum Ht for the two leading jets
+ -1  = ht3max ! maximum Ht for the three leading jets
+ -1  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+   0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ .true. = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0   = xetamin ! minimum rapidity for two jets in the WBF case
+ 0   = deltaeta ! minimum rapidity for two jets in the WBF case
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1    =  ktdurham
+ 0.4  =  dparameter
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
```

Comment:
You should do jet matching with some xqcut applied

Reply:
I changed this value to 30 after seeing it elsewhere. Is there a recommended value for this or a way to determine a proper value?

Reply:
For concrete study, you should try different xqcuts and qcuts (Pythia parameter) and make a DJR plot, and see if the distributions are smooth.

Reply:
I've attached several DJR plots with the xqcut and qcut variables in the names. The xqcut=30, qcut=45 plot looks the smoothest to me, but "smooth" is a rather vague term so perhaps I don't know what I'm looking for. Any input would be appreciated.

[djrs_10_30.pdf](https://github.com/cms-sw/genproductions/files/5934483/djrs_10_30.pdf)
[djrs_30_45.pdf](https://github.com/cms-sw/genproductions/files/5934484/djrs_30_45.pdf)
[djrs_30_60.pdf](https://github.com/cms-sw/genproductions/files/5934485/djrs_30_60.pdf)
[djrs_60_90.pdf](https://github.com/cms-sw/genproductions/files/5934486/djrs_60_90.pdf)
[djrs_200_300.pdf](https://github.com/cms-sw/genproductions/files/5934487/djrs_200_300.pdf)
[djrs_10_15.pdf](https://github.com/cms-sw/genproductions/files/5934488/djrs_10_15.pdf)

Reply:
Sorry for the late reply, 30-45 and 60-90 look best to me. 30-60 seems like the most dangerous choice (dip at 1.5).

Reply:
Do I need to create a pythia card for the qcut parameter? Or can I add this to my customaizecards.dat?

Reply:
customizecards are for ME level (=madgraph) so you should specify your qcuts in the hadronizer fragments